rules/selector.js

```diff
@@ -21,6 +21,18 @@ module.exports = {
   'selector-descendant-combinator-no-non-space': true,
   // Specify a pattern for id selectors.
   'selector-id-pattern': /^[A-Z][a-zA-Z]+$/,
+  // Limit the number of attribute selectors in a selector. <FEEDBACK>
+  'selector-max-attribute': null,
+  // Limit the number of classes in a selector. <FEEDBACK>
+  'selector-max-class': null,
+  // Limit the number of combinators in a selector. <FEEDBACK>
+  'selector-max-combinators': null,
+  // Limit the number of id selectors in a selector. <FEEDBACK>
+  'selector-max-id': null,
+  // Limit the number of type in a selector. <FEEDBACK>
+  'selector-max-type': null,
+  // Limit the number of universal selectors in a selector. <FEEDBACK>
+  'selector-max-universal': 2,
```

Comment:
These seem reasonable to me. The only one I'm not sure about is max type being 0. I know we really try to avoid these but things like styling default links may still call for it.